examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -157,8 +94,73 @@ spec:
         # edpm firewall, change the allowed CIDR if needed
         edpm_sshd_configure_firewall: true
         edpm_sshd_allowed_ranges:
-          - 192.168.122.0/24 # CHANGEME
+          - 192.168.122.0/24
         # SELinux module
         edpm_selinux_mode: enforcing
         # SRIOV settings
-        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov-phy4:eno4' # CHANGEME
+        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: 'sriov-phy4:eno4'
+    nodes:
+      edpm-compute-0:
+        ansible:
+          ansibleHost: 192.168.122.100
+        hostName: edpm-compute-0
+        networks:
+        - defaultRoute: true
+          fixedIP: 192.168.122.100
+          name: CtlPlane
+          subnetName: subnet1
+        - name: InternalApi
+          subnetName: subnet1
+        - name: Storage
+          subnetName: subnet1
+        - name: Tenant
+          subnetName: subnet1
+      edpm-compute-1:
+        ansible:
+          ansibleHost: 192.168.122.101
+        hostName: edpm-compute-1
+        networks:
+        - defaultRoute: true
+          fixedIP: 192.168.122.101
+          name: CtlPlane
+          subnetName: subnet1
+        - name: InternalApi
+          subnetName: subnet1
+        - name: Storage
+          subnetName: subnet1
+        - name: Tenant
+          subnetName: subnet1
+    services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - reboot-os
+    - libvirt
+    - ovn
+    - nova-custom-sriov
+    - neutron-sriov
+    - neutron-metadata
+  nova:
+    compute:
+      conf: |
+        [compute]
+        cpu_shared_set = 0-3,24-27
+        cpu_dedicated_set = 8-23,32-47
+        reserved_host_memory_mb = 4096
+    migration:
+      ssh_keys:
+        private: |
+          CHANGEME4
+          CHANGEME4
+          CHANGEME4
+        public: |
+          CHANGEME5
+          CHANGEME5
+          CHANGEME5
```

Comment:
we are currently expecting to use a single ssh key for all host for migration 

so we should not have 3 here.

Reply:
Sorry, the format I have chosen here for the `CHANGEME` is confusing.  I used multiple lines merely to draw the eye to the field so that the user knows to change it.  I will switch all of these to use a single `CHANGEME` to avoid giving the impression that multiple keys are expected.

Reply:
That's a good idea, but I'm not sure `kustomize` natively supports pulling in a file and injecting it into a template.  It looks like maybe you could write a plugin to do that, though, so perhaps we can add such a thing as a Jira item to follow-up upon: https://stackoverflow.com/questions/68447251/is-there-a-way-to-have-kustomize-merge-the-content-of-a-file-into-a-yaml-file.

Reply:
Let's inject that (single) key data via the values file for now. If you want to write a KRM function to pick it up via a path then let's do that in a separate PR. This patch doesn't need to block on that however.

Note also for fully automated CI jobs that values.yaml will be provided by the ci-framework.

After this merges I'd like to apply the same pattern for EDPM kustomize to VA1.

Reply:
ya this is not a blocker. i tough we were expecting 3 keys which wont work with what currently automated for cold/live migration

inlining the key is less secure but it wont break anything so this is not a blocker we can follow up once the toolign matures more